src/components/Form/index.js

```diff
@@ -48,19 +48,11 @@ export default class Form extends Component {
       }
 
       findNodesinDOM(props.children, ...formComponentNames)
-      .forEach(node => {
+      .forEach((node) => {
          this.state.data[node.props.name] = {
-<<<<<<< Updated upstream
-            value: node.props.value,
-            meta: {
-               error: null,
-               touched: false
-            }
-=======
             value: node.props.value || '',
```

Comment:
Ah I see. I think it was just throwing an error for me so I set it back to empty string. My bad.